docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -74,101 +74,230 @@ ml_train_model_construct(&model);
 // Compile model. This freezes model and afterwards the model cannot be modified.
```

Comment:
// Compile model - This freezes the model and afterward, the model cannot be modified.

Reply:
Done
-> // Compile model - This freezes the model. Afterwards the model cannot be modified.